docs/concepts/work-pools.md

```diff
@@ -154,6 +156,42 @@ Inspect the work pool:
 
 On success, the command returns the details of the newly created work pool, which can then be used to start agents that poll this pool for work or perform additional configuration of the pool.
 
+#### Base Job Template
+
+Each work pool has a base job template that allows the customization of the behavior of the worker executing flow runs from the work pool. 
+
+Each worker type is configured with a default base job template, making it is easy to get started with a work pool. The default base template defines fields that can be defined either as defaults on the work pool or defined on a deployment.
+
+For example, if we create a `process` work pool via the CLI:
+
+```bash
+$ prefect work-pool create --type process above-ground
+```
+
+We see these configuration options available in the Prefect UI:
+![process work pool configuration options](../img/ui/process-work-pool-config.png)
+
+
+For a `process` work pool, we can configure the option to set environment variables for spawned processes, set the working directory to execute flows, and control whether or not the flow run output is streamed to workers' standard output.
+
+You can override each of these attributes on a per-deployment basis. When deploying a project, you can specify these overrides in the `work_pool.job_variables` section of a `deployment.yaml`.
+
+If we wanted to turn off streaming output for a specific deployment, we could add the following to our `deployment.yaml`:
+
+```yaml
+work_pool:
+    name: above-ground  
+    job_variables:
+        stream_output: false
+```
+
+!!! tip "Advanced Customization of the Base Job Template"
+    For advanced use cases, users can fully customize the base job template of a work pool via the 'Advanced' tab in the work pool create and edit forms. 
+    
+    Advanced customization is useful for worker types that can support a high degree of customization. For example, the `kubernetes` worker type allows users to specify a custom job template that can be used to configure the manifest that workers use to create jobs for flow execution. 
```

Comment:
I think we can say more here - something like

Advanced customization in various scenarios: 

- 
- if there are 
```suggestion
    Advanced customization is useful anytime the underlying infrastructure supports a high degree of customization; in these scenarios a work pool job template allows you to expose a minimal and easy-to-digest set of options to deployment authors.  Additionally, these options are the _only_ customizable aspects for deployment infrastructure, which can be useful for restricting functionality in secure environments. For example, the `kubernetes` worker type allows users to specify a custom job template that can be used to configure the manifest that workers use to create jobs for flow execution. 
```